ruby/google-protobuf.gemspec

```
@@ -29,18 +29,17 @@ Gem::Specification.new do |s|
       ext/google/protobuf_c/extconf.rb
       ext/google/protobuf_c/Rakefile
     ]
-    s.add_development_dependency "rake-compiler-dock", "= 1.2.1"
+    s.add_development_dependency "rake-compiler-dock", ">= 1.7.0"
   end
   s.required_ruby_version = '>= 3.0'
+  s.add_dependency "bigdecimal"
   # TODO: evaluate removing Rakefile and moving logic to extconf.rb, so that we
   # can remove this runtime dependency on rake. See the discussion here for
   # more details:
   # https://github.com/protocolbuffers/protobuf/pull/15203
   s.add_dependency "rake", ">= 13"
-  s.add_development_dependency "ffi", "~>1"
+  s.add_development_dependency "ffi", ">= 1.17.0"
```

```suggestion
  s.add_development_dependency "ffi", ">= 1.17.1"
```
That will get you the changes from https://github.com/ffi/ffi/pull/1133 too.